<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>qwerty</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(238, 238, 236);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_image">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>340</width>
      <height>520</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_qwerty">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 50pt &quot;Waree&quot;;</string>
    </property>
    <property name="text">
     <string>Qwerty</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_login">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Waree&quot;;</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_user">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="placeholderText">
     <string>用户名或邮箱</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_passwd">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_login">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>390</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Log in</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_remember">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>记住密码</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="command_signup">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>460</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>New to us? Join now!</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
